internal/pkg/cli/env_deploy.go

```diff
@@ -267,12 +271,8 @@ After fixing the deployment, you can:
 	return fmt.Errorf("deploy environment %s: %w", o.name, err)
 }
 
-func environmentManifest(envName string, rawMft []byte, transformer interpolator) (*manifest.Environment, error) {
-	interpolated, err := transformer.Interpolate(string(rawMft))
-	if err != nil {
-		return nil, fmt.Errorf("interpolate environment variables for %q manifest: %w", envName, err)
-	}
-	mft, err := manifest.UnmarshalEnvironment([]byte(interpolated))
+func environmentManifest(envName string, rawMft []byte) (*manifest.Environment, error) {
```

Comment:
maybe we should now rename this func:
```suggestion
func unmarshalAndValidate(envName string, interpolated []byte) (*manifest.Environment, error) {
```

or, leave it as `environmentManifest`, but move 
```go
rawMft, err := o.ws.ReadEnvironmentManifest(o.name)
	if err != nil {
		return fmt.Errorf("read manifest for environment %q: %w", o.name, err)
	}
	interpolated, err := o.newInterpolator(o.appName, o.name).Interpolate(string(rawMft))
	if err != nil {
		return fmt.Errorf("interpolate environment variables for %q manifest: %w", o.name, err)
	}
```
into `environmentManifest` as well, and change its signature to ⬇️  so that we don't repeat ourselves in `env_deploy.go` and `env_package.go`.
```go
func environmentManifest(envName string) (mft *manifest.Environment, interpolated string, error)
```

I am not in favor of one or the other!

Reply:
Yeah the only reason why im repeating more myself is because i don't like the idea for having multiple return value. But it's less a concern for private func i guess